app/models/user.rb

```diff
@@ -9,6 +10,20 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  def authenticated? remember_token
+    return false if remember_digest.nil?
```

Comment:
```suggestion
    return unless remember_digest
```
1. Tìm hiểu method nil?, blank?, empty? ...

Reply:
okie anh